doc/sphinxman/source/glossary_psivariables.rst

```diff
@@ -120,9 +120,27 @@ PSI Variables by Alpha
    energy [H] and correlation correction components [H] for the compound
    method requested through cbs().
 
+.. psivar:: CC ROOT n DIPOLE X
```

Comment:
I don't know if `follow_root` would do that for the cc codes, unless it is done elsewhere. And yes the `DIPOLE -> QUADRUPOLE` switch is correct. I missed that.